sources/api/apiserver/src/server/ephemeral_storage.rs

```diff
@@ -118,10 +118,16 @@ pub fn bind(variant: &str, dirs: Vec<String>) -> Result<()> {
         _ => format!("{}{}", RAID_DEVICE_DIR, RAID_DEVICE_NAME),
     };
 
+    // Normalize input by trimming trailing "/"
+    let normalized_dirs: Vec<String> = dirs
```

Comment:
This is a case where I'd just create a new local with the same name to shadow the previous binding and ensure that it's not subsequently used by mistake:
```suggestion
    let dirs: Vec<String> = dirs
```

Otherwise, we have to keep the context in mind that "dirs is unsafe" and "normalized_dirs is safe" for subsequent uses in this function.